### PR TITLE
Added CREDIT_CARD test case

### LINKED ISSUE #15: Detect CREDIT_CARD
Considering that credit card information is classified as personal identifiable information, it is critical to identify and safeguard such information within today's digital landscape. Credit card information, including card numbers, expiration dates, and CVV codes, is highly sought out by cybercriminals for its potential to facilitate unauthorized transactions and identity theft.  Data breaches concerning credit card information primarily occur through sophisticated cyberattacks exploiting vulnerabilities in digital systems and networks through malware, hacking, and social engineering techniques.

It is also worth noting the direct connection that credit card usage has to the contribution of the credit score that ultimately determine ones ability to borrow, buy, or rent just about any desired asset. Therefore, strategic measures must be implemented to protect this sensitive data and prevent it from falling into the wrong hands.

For more insight, check out resource [1](https://blog.717cu.com/resources/education/financial-education-blog/important-credit-card-safety-tips-to-protect-your-personal-information#:~:text=It's%20important%20to%20be%20proactive,hunt%20for%20passwords%20and%20PINs.) and [2](https://listings.pcisecuritystandards.org/documents/PCI_DSS-QRG-v3_2_1.pdf) for more information.

### PR DESCRIPTION
**Pull request by emaniem**

### CODE REVIEW

File: test_team3.py [hunk 1]
```diff
@@ -16,3 +16,39 @@ def test_anonymize_nrp(self):
...
+        actual_result = anonymize_text(test_string, ['CREDIT_CARD'])
+        self.assertEqual(expected_result,actual_result)
+
+        #Negative Test Case - Credit Card Number too long: beyond 19 digits
+        test_cc = '1232 8324 3453 2353 6575 2344' 
+        test_string = 'My credit card number is ' + test_cc
+        expected_result = 'My credit card number is 1232 8324 3453 2353 6575 2344'
+        actual_result = anonymize_text(test_string, ['CREDIT_CARD'])
+        self.assertEqual(expected_result,actual_result)
+
```
Comment (shymik01) [inline test_team3.py]: Great , What about a negative test case for trying to enter a non numeric character ?

**Review by shymik01 — CHANGES_REQUESTED** [review]
Negative case for  user entering a Non-number

File: test_team3.py (hunk 1 above)
Comment (emaniem) [reply test_team3.py]: Change have been made. Thank you for the comment.

**Review by shymik01 — APPROVED** [review]
looks great !

**Review by Rahjion — APPROVED** [review]
Looks good

**Review by claesmk — APPROVED** [review]
Corrected the merge issues looks good now

**Comment by claesmk** [comment]
@emaniem I'm merging this for you

**Comment by claesmk** [comment]
@emaniem reminder to delete your development branch 15-detect-credit-card

**Pull request merged**